### PR TITLE
FT3: build v3b version of the geometry by default

### PR DESCRIPTION
**Pull request by plariono**
This PR is to build the FT3 geometry of v3b version from the LOI. Currently it builds the v1.

### CODE REVIEW

File: Detectors/Upgrades/ALICE3/FT3/simulation/src/Detector.cxx [hunk 1]
```diff
@@ -221,6 +221,55 @@ void Detector::buildFT3V1()
...
+    {34., .5, 3., 0.1f * layersx2X0},
+    {77., 5.0, 35., layersx2X0},
+    {100., 5.0, 35., layersx2X0},
+    {122., 5.0, 35., layersx2X0},
+    {150., 5.5, 100., layersx2X0},
+    {180., 6.6, 100., layersx2X0},
+    {220., 8.1, 100., layersx2X0},
+    {279., 10.2, 100., layersx2X0},
+    {340., 12.5, 100., layersx2X0},
+    {400., 14.7, 100., layersx2X0}};
```
Comment (mconcas) [inline Detectors/Upgrades/ALICE3/FT3/simulation/src/Detector.cxx]:
```suggestion
    {150., 5.5, 80.f, layersx2X0},
    {180., 6.6, 80.f, layersx2X0},
    {220., 8.1, 80.f, layersx2X0},
    {279., 10.2, 80.f, layersx2X0},
    {340., 12.5, 80.f, layersx2X0},
    {400., 14.7, 80.f, layersx2X0}};
```

Radius needs to be adjusted to current barrel max radius that is 80cm.

**Comment by mconcas** [comment]
Thanks @plariono , please find one single comment, you can merge the change from web if you want.

**Comment by plariono** [comment]
ciao @mconcas, thank you for the fix. In fact the outer radius of 80 cm corresonds to the version **v3b**, so I changed also the naming.

**Comment by mconcas** [comment]
@plariono as soon as you fix the code formatting, we can have CI go through.

**Review by mconcas — APPROVED** [review]

**Comment by plariono** [comment]
Can this go in please @mconcas ?

**Review by mconcas — APPROVED** [review]

**Pull request merged**